docs/pages/machine-workload-identity/machine-workload-identity.mdx

```diff
@@ -5,7 +5,7 @@ labels:
  - mwi
 ---
 
-Teleport Zero Trust Access & Flexible Workload Identity offers two complementary types of identity for non-human entities in your infrastructure:
+Teleport Machine & Workload Identity offers two complementary sets of functionality for non-human entities in your infrastructure:
```

Comment:
```suggestion
Teleport Machine & Workload Identity offers two complementary sets of capabilities for non-human entities in your infrastructure:
```
Since we don't typically say that a product has a set of functionality.